bot/exts/evergreen/bookmark.py

```diff
@@ -88,11 +89,16 @@ async def send_reaction_embed(
     async def bookmark(
         self,
         ctx: commands.Context,
-        target_message: WrappedMessageConverter,
+        target_message: t.Optional[WrappedMessageConverter],
         *,
         title: str = "Bookmark"
     ) -> None:
         """Send the author a link to `target_message` via DMs."""
+        if not target_message:
+            if not ctx.message.reference:
+                raise commands.UserInputError("You must either provide a valid message to bookmark, or reply to one.")
+            target_message = await ctx.channel.fetch_message(ctx.message.reference.message_id)
```

Comment:
Nitpick 🙂 
```suggestion
            target_message = await ctx.fetch_message(ctx.message.reference.message_id)
```
Then again, [`Message.resolved`](https://discordpy.readthedocs.io/en/latest/api.html#discord.MessageReference.resolved) exists, but I'm not sure how reliable that is.

Reply:
Actually message.reference.resolved doesnt exist because lance uses 1.5.1 not 1.6